wails/wails_handlers.go

```diff
@@ -92,6 +92,28 @@ func (app *WailsApp) WailsRequestRouter(route string, method string, body string
 		}
 	}
 
+	btcRateRegex := regexp.MustCompile(
```

Comment:
all this code can be removed, we do not need to pass the currency from the frontend